### DISCUSSION TITLE
One Twilio Theme for Support Centers

### DISCUSSION THREAD

**Discussion by tonge495** [twilio-labs/paste#3096] [Office hours] [answered]
### Background

Hi,

I'm Eileen.  I am the product designer for the Services Operations team, which focuses on providing technology solutions for our internal support teams.  One of the projects I am working on is the One Support Center, which is an initiative to consolidate the current Support Center experiences (e.g. [Twilio](https://support.twilio.com/hc/en-us), [Sendgrid](https://support.sendgrid.com/hc/en-us), and [Authy](https://support.authy.com/hc/en-us)) into one cohesive experience.  In the initial stage of this project, we are trying to create a simple end to end experience from when a user searches for a knowledge article to when they submit a support ticket.  We are expected to deliver on the initial experience to our stakeholders within the next 45/60 days.

I would like to review our current mockups for this flow, as this is the first time our team is using the One Twilio Theme for net new functionality.  I would primarily like to get some visual design feedback to see if we are on the right track with utilizing the theme appropriately.

- Link to Figma [mockups](https://www.figma.com/file/gqSvdYJnFoUZRwJFWPSm9i/One-Support-Center?node-id=236%3A138456&t=bUdSm5ozJo2y2lrP-1)

Looking forward to chatting, thank you!

### Are you looking for visual design support?

- [X] Yes

**Comment by serifluous** [comment]
Hey @tonge495 ! Thanks so much for signing up for office hours.

Just a note— @libertycarlin , our visual product designer will be OOO for office hours this week. She'll provide feedback in your Figma link async, and we'd still be super happy to see you in office hours regardless to talk through initial feedback and other questions you might have about using the new design language/theme.

**Reply by tonge495** [reply]
Sounds good, happy to share what I have!

Just as an FYI, as we are on a tight deadline with the implementation, I would prefer if we are able to resolve the majority of the design feedback during office hours this week.  If it's a small design tweak, then that is still something that I can probably accommodate  past this week but otherwise, it needs to go to the developers fairly soon.

**Comment by SiTaggart** [comment, marked as answer]
Thanks for coming to Office Hours!

The [recording](https://www.loom.com/share/166548500fa441829a59c83a4358b287) is available and password can be found the help design system Slack channel.
